write.cc

```diff
@@ -5,43 +5,52 @@
 #include <string>
 #include "messages.pb.h"
 
-using namespace std;
+//using namespace std;
+
 
 int main(int argc, char *argv[]){
     // Verify the version of the library
     GOOGLE_PROTOBUF_VERIFY_VERSION;
 
     int x, y, vel_x, vel_y;
-    FILE *fout;
-    string output;
+    //FILE *fout;
+    ::std::string output;
 
     if (argc != 2) {
-        cerr << "Usage: " << argv[0] << " file_to_store" << endl;
+        ::std::cerr << "Usage: " << argv[0] << " file_to_store" << ::std::endl;
         return -1;
-    }
+        }
 
     swarm::Data data;
 
-    cout << "Enter x: ";
-    cin >> x;
+    ::std::cout << "Enter x: ";
+    ::std::cin >> x;
     data.set_x(x);
-    cout << "Enter y: ";
-    cin >> y;
+    ::std::cout << "Enter y: ";
+    ::std::cin >> y;
     data.set_y(y);
-    cout << "Enter vel_x: ";
-    cin >> vel_x;
+    ::std::cout << "Enter vel_x: ";
+    ::std::cin >> vel_x;
     data.set_vel_x(vel_x);
-    cout << "Enter vel_y: ";
-    cin >> vel_y;
+    ::std::cout << "Enter vel_y: ";
+    ::std::cin >> vel_y;
     data.set_vel_y(vel_y);
 
     if (!data.SerializeToString(&output)) {
-        cerr << "Failed to write to the address book." << endl;
+        ::std::cerr << "Failed to write to the address book." <<::std::endl;
         return -1;
     }
 
-    fout = fopen(argv[1], "w");
-    fprintf(fout, "%s", output.c_str());
+    ::std::ofstream obj(argv[1], ::std::ios::out);
+    if(!obj)
+        {
```

Comment:
In the whole of the code, we have followed the format of `if () {` and it would be good to follow the same.

Note the space after `if` too